openapi-codegen/core/src/main/scala/sttp/tapir/codegen/SchemaGenerator.scala

```diff
@@ -152,40 +152,40 @@ object SchemaGenerator {
   ): Set[String] = referenceCandidate match {
     case ref: OpenapiSchemaRef =>
       val stripped = ref.stripped
-      // in this case, we have a chain of referrences from referrent -> [...maybeRefs] -> referrent, creating a mutually-recursive loop
-      if (stripped == referrent) maybeRefs.toSet
+      // in this case, we have a chain of references from referent -> [...maybeRefs] -> referent, creating a mutually-recursive loop
+      if (stripped == referent) maybeRefs.toSet
       // if already checked, skip
       else if (checked contains stripped) Set.empty
       // else add the ref to 'maybeRefs' chain and descend
       else {
         allSchemas
           .get(ref.stripped)
-          .map(getReferencesToXInY(allSchemas, referrent, _, checked + stripped, maybeRefs :+ stripped))
+          .map(getReferencesToXInY(allSchemas, referent, _, checked + stripped, maybeRefs :+ stripped))
           .toSet
           .flatten
       }
-    // these types cannot contain a referrence
+    // these types cannot contain a reference
     case _: OpenapiSchemaSimpleType | _: OpenapiSchemaEnum | _: OpenapiSchemaConstantString => Set.empty
     // descend into the sole child type
-    case OpenapiSchemaArray(items, _) => getReferencesToXInY(allSchemas, referrent, items, checked, maybeRefs)
-    case OpenapiSchemaNot(items)      => getReferencesToXInY(allSchemas, referrent, items, checked, maybeRefs)
-    case OpenapiSchemaMap(items, _)   => getReferencesToXInY(allSchemas, referrent, items, checked, maybeRefs)
+    case OpenapiSchemaArray(items, _) => getReferencesToXInY(allSchemas, referent, items, checked, maybeRefs)
+    case OpenapiSchemaNot(items)      => getReferencesToXInY(allSchemas, referent, items, checked, maybeRefs)
+    case OpenapiSchemaMap(items, _)   => getReferencesToXInY(allSchemas, referent, items, checked, maybeRefs)
     // descend into all child types
-    case OpenapiSchemaOneOf(items, _) => items.flatMap(getReferencesToXInY(allSchemas, referrent, _, checked, maybeRefs)).toSet
-    case OpenapiSchemaAllOf(items)    => items.flatMap(getReferencesToXInY(allSchemas, referrent, _, checked, maybeRefs)).toSet
-    case OpenapiSchemaAnyOf(items)    => items.flatMap(getReferencesToXInY(allSchemas, referrent, _, checked, maybeRefs)).toSet
+    case OpenapiSchemaOneOf(items, _) => items.flatMap(getReferencesToXInY(allSchemas, referent, _, checked, maybeRefs)).toSet
+    case OpenapiSchemaAllOf(items)    => items.flatMap(getReferencesToXInY(allSchemas, referent, _, checked, maybeRefs)).toSet
+    case OpenapiSchemaAnyOf(items)    => items.flatMap(getReferencesToXInY(allSchemas, referent, _, checked, maybeRefs)).toSet
     case OpenapiSchemaObject(kvs, _, _) =>
-      kvs.values.flatMap(v => getReferencesToXInY(allSchemas, referrent, v.`type`, checked, maybeRefs)).toSet
+      kvs.values.flatMap(v => getReferencesToXInY(allSchemas, referent, v.`type`, checked, maybeRefs)).toSet
```

Comment:
a bunch of completely unrelated spelling/grammar fixes that I'd forgotten I had committed in my local branch. Does no harm I suppose 😅